tools/cockpit.spec

```diff
@@ -124,6 +124,7 @@ if [ -n "%{patches}" ]; then
 	git add -f . && git commit -a -q -m "Base" && git tag -a initial --message="initial"
 	git am --whitespace=nowarn %{patches}
 	touch -r $(git diff --name-only initial..HEAD) .git
+	touch configure.ac aclocal.m4 configure Makefile.am Makefile.in
```

Comment:
configure.ac and Makefile.am are original source and really should *not* be touched.

This will silently break patches which forget to update the autotools generated files along with the source files. Do we really want that?